polkadot/runtime/parachains/src/runtime_api_impl/mod.rs

```diff
@@ -25,5 +25,8 @@
 //! 1. Bump the version of the stable module (e.g. `v2` becomes `v3`)
 //! 2. Move methods from `vstaging` to `v3`. The new stable version should include all methods from
 //!    `vstaging` tagged with the new version number (e.g. all `v3` methods).
+
 pub mod v5;
+pub mod v6;
+pub mod v7;
```

Comment:
why do we have multiple versions and not just v7? like we do with primitives

Reply:
Yes makes sense to merge into a single v7.

Also, should I align primitives and runtime API versions (primitives are currently v6)?

Reply:
they don't have to align. e.g. we could add a new runtime API method, but not introduce a new primitive type